deployments/fpga_plugin/base/intel-fpga-plugin-daemonset.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: fpgadeviceplugin
+  name: intel-fpga-plugin
```

Comment:
let's split these `intel-fpga-plugin` YAML changes to a separate commit so that the `suite_test.go` patch goes away.